src/main/java/de/hysky/skyblocker/skyblock/dwarven/CrystalsLocationsManager.java

```diff
@@ -131,20 +178,59 @@ private static void registerWaypointLocationCommands(CommandDispatcher<FabricCli
 
     protected static Text getSetLocationMessage(String location, BlockPos blockPos) {
         MutableText text = Constants.PREFIX.get();
-        text.append(Text.literal("Added waypoint for "));
-        Color locationColor = WAYPOINT_LOCATIONS.get(location).color;
-        text.append(Text.literal(location).withColor(locationColor.getRGB()));
-        text.append(Text.literal(" at : " + blockPos.getX() + " " + blockPos.getY() + " " + blockPos.getZ() + "."));
+        text.append(Text.translatable("skyblocker.config.mining.crystalsWaypoints.addedWaypoint"));
+        int locationColor = WAYPOINT_LOCATIONS.get(location).getColor();
+        text.append(Text.literal(location).withColor(locationColor));
+        text.append(Text.literal(" ").append(Text.translatable("skyblocker.config.mining.crystalsWaypoints.addedWaypoint.at")));
+        text.append(Text.literal(" : " + blockPos.getX() + " " + blockPos.getY() + " " + blockPos.getZ() + "."));
 
         return text;
     }
 
-    private static Text getLocationInputText(String location) {
+    /**
+     * Creates a formated text with a list of possible places to add a waypoint for
+     *
+     * @param location       the location where the waypoint will be created
+     * @param excludeUnknown if the "Unknown" location should be available to add
```

Comment:
Small nitpick, the `"Unknown"` location could be represented with a link to the related enum entry, like so: 
```
 * @param excludeUnknown if the {@link de.hysky.skyblocker.skyblock.dwarven.MiningLocationLabel.CrystalHollowsLocationsCategory#UNKNOWN Unknown} location should be available to add
```